content/efcore/getting-started/samples/loading-related-data/include-method/Program.cs

```diff
@@ -12,7 +12,7 @@ public static void Main()
                 .Include(b => b.Author)
                 .Single(b => b.Id == 1);
                 
-            Console.WriteLine("{0} - {1} {2}", book.Title, book.Author.FirstName, book.Author.LastName);
+			Console.WriteLine("Included author with book - \nId: {0}\n Title: {1}\n Genre: {2}\n Publication Year: {3}\nAuthor: {4} {5}", book.Id, book.Title, book.Genre, book.PublicationYear, book.Author.FirstName, book.Author.LastName);
```

Comment:
Use multiple `Console.WriteLine` statements instead of the newline characters. It will make the code more readable.